pylmod/client.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
"Starter"?  Are you recommending that I include some lines from that file into this one?